pubsub/cloud-client/publisher.py

```diff
@@ -183,17 +183,24 @@ def publish_messages_with_batch_settings(project_id, topic_name):
     # Configure the batch to publish as soon as there is one kilobyte
     # of data or one second has passed.
     batch_settings = pubsub_v1.types.BatchSettings(
-        max_bytes=1024, max_latency=1  # One kilobyte  # One second
+        max_bytes=1024,
```

Comment:
Can we put the default setting in a comment here? Some users have expressed that this would be useful.

Reply:
Good idea. Done!